docs/layers/grid-layer.md

```diff
@@ -0,0 +1,58 @@
+# GridLayer
+
+The GridLayer is a generic layer that can render heatmap. It takes latitude longitude delta
+of the cells, and the top left coordinate each cells. grid can be 3d when pass in a height
+and set enable3d to true
+
+<div align="center">
+  <img height="300" src="/demo/src/static/images/grid-layer.png" />
+</div>
+
+    import {GridLayer} from 'deck.gl';
+    
+Inherits from all [Base Layer](/docs/layers/base-layer.md) properties.
+
+## Layer-specific Properties
+
+##### `latDelta` (Number, optional)
```

Comment:
If we want this layer to support meters mode, it is best to not associate the name of these props with `lng` and `lat`.  This could be changed now to e.g. `cellWidth` and `cellHeight` or handled during API audit.